001-force-modules-rpath-using-linker-flags.patch

This patch is needed to ensure all rpath are properly assigned
to modules shared objects as otherwise only default cmake rpath
is being used while LDFLAGS extra rpath not being considered

--- modules/CMakeLists.txt-orig	2025-07-01 21:17:15.000000000 +0000
+++ modules/CMakeLists.txt	2025-07-09 11:18:52.139294029 +0000
@@ -17,6 +17,8 @@
 # This is not recommended, but whatever.
 file(GLOB all_modules LIST_DIRECTORIES FALSE "*")
 
+set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} ${CMAKE_EXE_LINKER_FLAGS}")
+
 function(add_cxx_module mod modpath)
 	znc_add_library("module_${mod}" MODULE "${modpath}")
 	set_target_properties("module_${mod}" PROPERTIES
